src/content/pt/2019/accessibility.md

```diff
@@ -0,0 +1,317 @@
+---
+part_number: II
+chapter_number: 9
+title: Acessibilidade
+description: Capítulo de acessibilidade do Almanaque da Web de 2019 cobrindo facilidade de leitura, mídia, facilidade de navegação e compatibilidade com tecnologias assistivas.
```

Comment:
>description: Capítulo de acessibilidade do Almanaque da Web de 2019 cobrindo facilidade de leitura, mídia, facilidade de navegação e compatibilidade com tecnologias assistivas.

@eduqg de acordo com [Translators'-Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Translators'-Guide), acredito que não possa ser traduzida.

>English product names (Web Almanac, HTTP Archive) should remain in English but can be adjusted slightly to local version (e.g. Le Web Almanac rather than The Web Almanac). For some languages this rule may be ignored if there are language specific character sets (e.g. Japanese might use katakana, hiragana, or kanj rather than English if that makes more sense).